gatsby/src/components/BackToTop/BackToTop.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState, useCallback } from "react";
+import ChevronUp from "@nice-digital/icons/lib/ChevronUp";
+import throttle from "lodash/throttle";
+
+import styles from "./BackToTop.module.scss";
+
+const scrollThrottle = 100;
+const pagesScrollThreshold = 1.5; // number of viewports to scroll before "back to top" appears
+
+interface BackToTopProps {
+	scrollTargetId?: string;
+}
+
+export const BackToTop: React.FC<BackToTopProps> = ({
+	scrollTargetId: scrollTo = "content-start",
+}: BackToTopProps) => {
+	const [isFixed, setIsFixed] = useState(false);
+
+	const handleScroll = useCallback(() => {
+		const footer: Element | null = document.querySelector("footer");
+
+		if (!footer) return;
+
+		const footerVisible: boolean =
+			window.scrollY + window.innerHeight >
+			document.body.scrollHeight - footer.clientHeight;
+
+		const scrolledDown: boolean =
+			window.scrollY > window.innerHeight * pagesScrollThreshold;
+
+		setIsFixed(!footerVisible && scrolledDown);
+	}, []);
+
+	useEffect(() => {
+		document.addEventListener("scroll", throttle(handleScroll, scrollThrottle));
+
+		return () => {
+			document.removeEventListener("scroll", handleScroll);
```

Comment:
This removal isn't a reference to the same function that got added, as it's now wrapped in throttle, so I don't think this cleanup will work